docs/commands.md

```diff
@@ -1095,6 +1095,7 @@ hal config features edit [parameters]
  * `--chaos`: Enable Chaos Monkey support. For this to work, you'll need a running Chaos Monkey deployment. Currently, Halyard doesn't configure Chaos Monkey for you; read more instructions here https://github.com/Netflix/chaosmonkey/wiki.
  * `--deployment`: If supplied, use this Halyard deployment. This will _not_ create a new deployment.
  * `--jobs`: Allow Spinnaker to run containers in Kubernetes and Titus as Job stages in pipelines.
+ * `--mine-canary`: Enable canary support. For this to work, you'll need a canary judge configured. Currently, Halyard does not configure canary judge for you.
```

Comment:
Any reason why this is called `mine-canary` now? That might get confusing when we open source the new canary judge, and users don't understand what the difference between mine and kayenta is.

Reply:
Talked out-of-band - this makes sense. Can we update settings.js too just so we don't get caught on this in the future?